bedrock/firefox/templates/firefox/hub/home-quantum.html

```diff
@@ -358,7 +358,14 @@ <h2 class="section-title">{{ _('Sync your devices') }}</h2>
         <div class="browse-for-good-section-content">
           <h2 class="section-title">{{ _('Browse for good') }}</h2>
           <p>
-          {% if l10n_has_tag('firefox-home-copy-update-20171110') %}
+          {% if l10n_has_tag('firefox-home-copy-update-20171114') %}
+            {% trans %}
+              Firefox is made by Mozilla, the non-profit champions of a healthy internet.
+              Mozilla also tackles issues like privacy, misinformation and trolling by
+              investing in fellowships, campaigns and new technologies designed to make
+              the internet healthier.
+            {% endtrans %}
+          {% elif l10n_has_tag('firefox-home-copy-update-20171110') %}
```

Comment:
We didn't expose strings for #5265 yet. You can get rid of the this section, and only leave new string and old one (with girls instead of women).

Reply:
👍